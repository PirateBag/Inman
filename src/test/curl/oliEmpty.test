Generate a report of all order line items.
Setup:
No order line items have been created

Expected output:
Empty report but no failures.
===== JSON ====
{ "idToSearchFor" : "-1" }