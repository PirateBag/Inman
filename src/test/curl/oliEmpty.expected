{
  "responseType" : "MULTILINE",
  "errors" : [ {
    "key" : 1,
    "message" : "No items were processed, either due to errors or no actionable inputs."
  } ],
  "data" : [ ]
}